### PR TITLE
Document audience-based token format

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### CODE REVIEW

File: docs/source/app-dev/authorization.rst [hunk 1]
```diff
@@ -157,14 +157,42 @@ the participant user management service *without* issuing new access tokens,
...
 
 User access tokens are `JWTs <https://datatracker.ietf.org/doc/html/rfc7519>`_ that follow the
-`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_ with a JSON payload of the following format.
+`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_. There are two
+different JSON encodings: An audience-based token format that relies
+on the audience field to designate that it is designed for a specific
+Daml participant and a scope-based audience format which relies on the
+scope field to designate. Both formats can be used interchangeably but
+if possible, use of the audience-based token format is recommend as it
+is compatible with a wider range of IAMs, e.g., Kubernetes does not
```
Comment (skisel-da) [inline docs/source/app-dev/authorization.rst:166]: Is it really Kubernetes problem or more like service mesh products like Istio, traefik, etc?

Comment (cocreature) [reply docs/source/app-dev/authorization.rst:166]: It is k8s, k8s can issue tokens itself but doesn’t let you set `scope`.

File: docs/source/app-dev/authorization.rst [hunk 2]
```diff
@@ -157,14 +157,42 @@ the participant user management service *without* issuing new access tokens,
 as would be required for the custom Daml claims tokens.
 
 User access tokens are `JWTs <https://datatracker.ietf.org/doc/html/rfc7519>`_ that follow the
-`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_ with a JSON payload of the following format.
+`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_. There are two
+different JSON encodings: An audience-based token format that relies
+on the audience field to designate that it is designed for a specific
+Daml participant and a scope-based audience format which relies on the
+scope field to designate. Both formats can be used interchangeably but
```
Comment (skisel-da) [inline docs/source/app-dev/authorization.rst:164]:
```suggestion
different JSON encodings: An audience-based token format that relies
on the audience field to designate that it is defined for a specific
Daml participant and a scope-based audience token format which relies on the
scope field to designate the purpose. Both formats can be used interchangeably but
```

I found this sentence difficult to understand so I tried to rephrase it... not sure mine reads better, but somehow I call for rephrasing ;)

File: docs/source/app-dev/authorization.rst [hunk 3]
```diff
@@ -157,14 +157,42 @@ the participant user management service *without* issuing new access tokens,
 as would be required for the custom Daml claims tokens.
 
 User access tokens are `JWTs <https://datatracker.ietf.org/doc/html/rfc7519>`_ that follow the
-`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_ with a JSON payload of the following format.
+`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_. There are two
+different JSON encodings: An audience-based token format that relies
+on the audience field to designate that it is defined for a specific
```
Comment (cocreature) [inline docs/source/app-dev/authorization.rst:162]:
```suggestion
on the audience field to specify that it is designated for a specific
```

**Review by skisel-da — APPROVED** [review]

**Pull request merged**